listenbrainz/db/user_timeline_event.py

```diff
@@ -158,6 +159,27 @@ def get_recording_recommendation_events_for_feed(user_ids: List[int], min_ts: in
         return [UserTimelineEvent(**row) for row in result.fetchall()]
 
 
+def get_user_timeline_events_by_id(id: int) -> UserTimelineEvent:
```

Comment:
If this method returns only one item the name probably shouldn't be plural:
```suggestion
def get_user_timeline_event_by_id(id: int) -> UserTimelineEvent:
```

Reply:
Pluralizing makes sense, thanks for catching it! :D